readme.md

```diff
@@ -20,14 +22,32 @@ configure :development do
 end
 ```
 
-If you want the php to be parsed on builds, too (as opposed to raw PHP generated for a server to run), just omit the `configure` block:
+If you want the PHP to be parsed on builds too (as opposed to raw PHP generated for a server to run), just omit the `configure` block:
 
 ```ruby
 activate :php
 ```
 
-The PHP will only be rendered on pages where the URL ends in `.php`...so if you want to see `index.php`, you'll have to actually stick `index.php` in your browser. It will however, let you do fancy stuff like files ending in `.php.erb`...so your ERB can generate some php output.
+Here is the list of settings with the default values:
 
-Also thanks to [Mariano](https://github.com/mcavallo), who contributed a significant chunk of code to get $_GET, $_POST, and other PHP global request variables working.
+```ruby
+activate :php do |config|
+  config.show_debug = true # Output debug info to console
```

Comment:
It's been a while since I've looked at this so maybe I'm just reading it incorrectly, but isn't this `false` by default, not true?